python/lsst/obs/hsc/hscMapper.py

```diff
@@ -192,7 +192,7 @@ def std_raw(self, item, dataId):
 
     def std_dark(self, item, dataId):
         exposure = self._standardizeExposure(self.calibrations['dark'], item, dataId, trimmed=False)
-        visitInfo = afwImage.makeVisitInfo(exposureTime=1.0)
+        visitInfo = afwImage.makeVisitInfo(exposureTime=1.0, darkTime=1.0)
```

Comment:
This looks fine. However, I am curious why darks don't already have correct exposure time (at least), a question that is out of scope, but left over from when I first added VisitInfo support to obs_subaru.